rust_src/crates/webrender/src/term.rs

```diff
@@ -844,7 +845,8 @@ extern "C" fn delete_frame(f: *mut Lisp_Frame) {
     display_info.get_inner().outputs.remove(&window_id);
 
     // Take back output ownership and destroy it
-    let _ = unsafe { Box::from_raw(output.as_rust_ptr()) };
+    let _ = unsafe { Box::from_raw(output.as_rust_ptr()).deinit() };
```

Comment:
rust's ownership will call `drop` trait function of a object automatically for us.